hub/views/accounts.py

```diff
@@ -98,3 +100,42 @@ def get_context_data(self):
         ).count()
 
         return context
+
+
+class AccountsCSV(PermissionRequiredMixin, ListView):
+    model = UserProperties
+    queryset = UserProperties.objects.order_by(F("last_seen").desc(nulls_last=True))
+    permission_required = "auth.edit_user"
+
+    def render_to_response(self, context, **response_kwargs):
+        response = HttpResponse(content_type="text/csv")
+        field_names = [
+            "name",
+            "organisation",
+            "email",
+            "date_joined",
+            "date_last_seen",
+            "email_confirmed",
+            "account_confirmed",
+            "is_active",
+            "is_staff",
+            "is_superuser",
+        ]
+        writer = csv.DictWriter(response, field_names)
+        writer.writeheader()
+        for userprops in self.object_list:
```

Comment:
if you have a `context_object_name = "users"` at the top of the class then this can use `context["users"]` instead of `self.object_list` which would, I think, be preferable.